desktop/apps/article/client/index.coffee

```diff
@@ -113,26 +116,23 @@ module.exports = class ArticleIndexView extends Backbone.View
 
   setupPromotedContent: =>
     if @article.get('channel_id') is sd.PC_ARTSY_CHANNEL
-      new Partner(id: @article.get('partner_ids')?[0]).fetch
-        error: => $('.articles-promoted').hide()
-        success: (partner) ->
-          new Profile(id: partner.get('default_profile_id')).fetch
-            error: => $('.articles-promoted').hide()
-            success: (profile) ->
-              $('#articles-show').prepend promotedTemplate
-                src: profile.bestAvailableImage()
-                name: partner.get('name')
-                href: profile.href()
-                type: profile.profileType()
+      send = method: 'post', query: partnerQuery(@article.get('partner_ids')?[0])
+      metaphysics send
+      .then (data) =>
+        $('#articles-show').prepend promotedTemplate
+          src: data.partner.profile.image.cropped.url
```

Comment:
I know when we paired on this we assumed these objects exist, but it might be a good to test with a partner that doesn't have a cropped image -- maybe that's a requirement on the profile, maybe not? Usually I add `?` if the API response doesn't guarantee fields.